osscameroon/yotas
# Add e2etest in ci

#### This pull request add end to end tests in our github action pipeline

**Why ?**
This will help us catch issues in the api before the PR gets merged

**How ?**
We first created a `docker-compose.test.yml` that contains every component necessary to run the end2end tests. 
And finally we have integrated the end to end test in out github workflow.

**Steps to verify:**
Locally you can run `make docker-e2etest` or if you want to see how it work on CI create a PR or have re-run this github action 
https://github.com/osscameroon/yotas/pull/73/checks?check_run_id=3692058419

#### Note this PR depends on #72 and should only be merged after its dependencies are merged
